dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/JsonPathSkill.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Linq;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Newtonsoft.Json;
```

Comment:
Why are we adding a depedency to Newtonsoft.Json to do this string.

Can't we rely on `System.Text.Json.Nodes` ?

Reply:
@adrianwyatt I sent a code snippet to you via chat last night -- take a look, as I think it does what you need with Sys.Text.Json

Reply:
I saw this thread when doing my initial research. I hesitate to use this solution since it's a custom language for parsing JSON. JsonPath is really the only standard in general use and I think it is important to stick with that for these foundational elements.